src/components/buttons/ButtonAdd.js

```diff
@@ -0,0 +1,28 @@
+import { useDispatch } from 'react-redux';
+import { addBook } from '../../redux/books/booksSlice';
+import books from '../ListBooks';
+
+const ButtonAdd = () => {
+  const dispatch = useDispatch();
+
+  const SubmitBook = (e) => {
+    const button = e.target;
+    const dataForm = button.parentElement;
+    const book = dataForm.querySelector('#bookTitle').value;
+    const author = dataForm.querySelector('#author').value;
+
+    if (book === '' || author === '') {
+      const message = 'Enter all the data please';
+      document.querySelector('.messagecont').textContent = message;
+    } else {
+      document.querySelector('.messagecont').textContent = '';
+      dispatch(addBook(books(book, author)));
+      dataForm.reset();
+    }
+  };
```

Comment:
- [ ] Great job on implementing the project as specified in the requirement! 👏 However, there is a minor issue 🐞 that we need to address and fix before moving on to the next milestone. To do that kindly consider the below suggestions: 
   - [ ] Kindly consider leveraging React's powerful feature called `useState` to manage the state of your books in your app, rather than relying on vanilla JavaScript. By using `useState`, you can simplify the state management process and make your code more maintainable and readable. It provides a straightforward and efficient way to handle state changes within functional components, improving the overall performance of your application.

   - [ ] Utilizing `useState` will also ensure that your component's state updates are handled correctly and efficiently, as React will take care of the necessary rendering optimizations. This way, you can focus on building the application's logic and user interface without worrying about the complexities of state management.

Reply:
Hi @Strangeal, thanks for your suggestions. Do you mean something like this?

![image](https://github.com/AndreaM2429/Bookstore/assets/119966922/c664f37a-62de-4b5b-b2c2-2fb8e98b264f)